tests/common/snappi_tests/traffic_generation.py

```diff
@@ -587,7 +588,8 @@ def verify_pause_frame_count_dut(duthost,
     on the DUT
 
     Args:
-        duthost (obj): DUT host object
+        rx_dut (obj): Ingress DUT host object
+        tx_dut (obj): Egress DUT host object
```

Comment:
pls reuse the same comment here too to indicate what traffic are expected on rx and tx ports.
# rx_dut is ingress DUT receiving packets.
# tx_dut is egress DUT transmitting packets and also receiving PFCs.